endpoints-control/src/main/java/com/google/api/control/ControlFilter.java

```diff
@@ -222,10 +218,8 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
     MethodRegistry.Info info = ConfigFilter.getMethodInfo(request);
     HttpServletRequest httpRequest = (HttpServletRequest) request;
     if (info == null) {
-      if (log.isLoggable(Level.FINE)) {
-        log.log(Level.FINE, String.format("no method corresponds to %s - skipping service control",
-            httpRequest.getRequestURI()));
-      }
+      log.atFine().log(
+          "no method corresponds to %s - skipping service control", httpRequest.getRequestURI());
```

Comment:
This is probably inexpensive enough to not worry about, but if you do end up using `lazy` or something like that, maybe use it here too.

Reply:
I'm not too worried about this one. Plus using lazy pretty ugly until we migrate to Java 8, which we will do and can revisit later.